docs/extend/modular-resources/create/_index.md

```diff
@@ -154,6 +152,14 @@ class MyBase(Base, Reconfigurable):
         self.left.set_power(power=((linear.y - angular.z) / sum), extra=extra, timeout=timeout)
         self.right.set_power(power=((linear.y + angular.z) / sum), extra=extra, timeout=timeout)
 
+    # set_velocity: unimplemented
+    async def set_velocity(self, linear: Vector3, angular: Vector3, *, extra: Optional[Dict[str, Any]] = None, timeout: Optional[float] = None, **kwargs):
+        pass
```

Comment:
prefer `raise NotImplementedError`, pass won't error out here when called which is sometimes more confusing

Reply:
Done!